Tests/SopraGameController.cpp

```diff
@@ -0,0 +1,50 @@
+//
+// Created by jonas on 30.04.19.
+//
+
+#include <gtest/gtest.h>
+#include "SopraGameController.h"
+#include "SopraGameModel.h"
+
+TEST(controller_test, getDistance) {
+    EXPECT_EQ(gameController::getDistance(gameModel::Position(0, 0), gameModel::Position(0, 0)), -1);
```

Comment:
Gerne in mehrere Tests auslagern, dann ist beim Testen sofort ersichtlich was nicht funktioniert wenn ein Test fehlschlägt

Reply:
Also einfach ganz viele von der Sorte?
TEST(controller_test, getDistance) {
    EXPECT_EQ(gameController::getDistance(gameModel::Position(0, 0), gameModel::Position(0, 0)), -1);
}

Reply:
Ja, dann die Namen irgendwie durchnummerieren und wenn dann die CI meint das der Test fehlschlägt wisst ihr schon ziemlich genau was das Problem ist ohne ein mal GDB zu starten.